distros/generate.sh

```diff
@@ -9,7 +9,13 @@ set -e
 #	or: ./generate.sh debian-newversion
 #		to create a new folder and a Dockerfile within it
 
-cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
+# `readlink -f` does not work on macOS.
+# To get `greadlink`, run `brew install coreutils`.
+if [ "$(uname)" == "Darwin" ]; then
```

Comment:
I meant GNU sed, but it sounds like you're implying that's upstream in dcos/dcos, not in dcos-docker. However, I was under the impression that the installer didn't work on Mac. Are you saying that GNU sed was the only blocker for that?